src/test/java/StringTest.java

```diff
@@ -0,0 +1,91 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    @Test
+    @DisplayName(",로 split을 했을 때 분리가 잘 되는지 확인")
+    void split_test_01() {
+        //given
+        String input = "1,2";
+        String[] expect = {"1", "2"};
+
+        //when
+        String[] result = input.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?